app/static/base.css

```diff
@@ -9,3 +9,41 @@ nav {
 .content-container {
     align-items: center;
 }
+
```

Comment:
Is there a reason for comment out as opposed to deleting?

Reply:
I'm going to go back and use these.

Reply:
I am going to be more strict about code since we are getting close to finalizing this project, and because of this I don't want to push dead code.

Reply:
If this is here and commented out by the meeting on April 20, we can just delete it then.

Reply:
I'm not merging dead code.